src/UI/Components/ServiceInstanceForm/Components/AutoCompleteInputProvider.tsx

```diff
@@ -35,7 +36,8 @@ export const AutoCompleteInputProvider: React.FC<Props> = ({
     kind: "GetServiceInstances",
     name: serviceName,
     filter,
-    pageSize: PageSize.from("100"),
+    pageSize: PageSize.from("250"),
```

Comment:
I don't think it's wise to change it by default to 250 (remember the performance issues from our last teambuilding. That page would display 150 more rows of loaded data, by default.)

Reply:
I understood that component as an autocomplete for i.e. relations so I thought it's fine, but I can revert it